Doc/whatsnew/3.13.rst

```diff
@@ -115,6 +115,9 @@ Removed
   are now removed. The items in those namespaces can be imported directly
   from :mod:`typing`. (Contributed by Sebastian Rittau in :gh:`92871`.)
 
+* Remove :class:`!webbrowser.MacOSX` class, deprecated in Python 3.11.
```

Comment:
```suggestion
* Remove the untested and undocumented :mod`webbrowser` :class:`!MacOSX` class,
  deprecated in Python 3.11.
  Use the :class:`!MacOSXOSAScript` class instead, introduced in Python 3.2.
```

* Briefly justify the change
* Mention its replacement
* Link to the module for reference
* Use the same "the"/complete sentence phrasing as the others